test/common/k8s/deploy/deploy.go

```diff
@@ -57,11 +96,15 @@ func CreateIdentityValidator(subscriptionID, resourceGroup, name, identityBindin
 
 	// Go template parameters to be translated in test/e2e/template/deployment.yaml
 	deployData := struct {
-		Name            string
-		IdentityBinding string
+		Name                     string
+		IdentityBinding          string
```

Comment:
Do we want to take namespace here as well ?

Reply:
We could. Right now all the E2E test pods are created in the default namespace.